src/main/java/com/axibase/tsd/client/HttpClientManager.java

```diff
@@ -17,9 +17,11 @@
 import com.axibase.tsd.model.system.ClientConfiguration;
 import com.axibase.tsd.network.PlainCommand;
 import com.axibase.tsd.query.QueryPart;
+import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.pool2.BasePooledObjectFactory;
 import org.apache.commons.pool2.PooledObject;
 import org.apache.commons.pool2.impl.DefaultPooledObject;
+import org.apache.commons.pool2.impl.DefaultPooledObject;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AUselessImportCheck)